apps/gp2-server/test/controllers/output.controller.test.ts

```diff
@@ -357,10 +357,17 @@ describe('outputs controller', () => {
 
       expect(result).toEqual(getOutputResponse());
 
-      const outputUpdateDataObject = getOutputUpdateDataObject();
-      expect(outputDataProviderMock.update).toBeCalledWith(outputId, {
-        ...outputUpdateDataObject,
-      });
+      const outputUpdateDataObject = {
+        ...getOutputUpdateDataObject(),
+        versions: ['1'],
+      };
+      expect(outputDataProviderMock.update).toHaveBeenCalledWith(
+        outputId,
+        {
+          ...outputUpdateDataObject,
+        },
+        { newVersion: undefined },
```

Comment:
what about testing the opposite scenario, when the newVersion field is populated?